lib/application/Backbone.js

```diff
@@ -1,7 +1,7 @@
 "use strict";
 
 var Backbone = require("backbone").noConflict();
-
+Backbone._ = require("lodash");
 module.exports = Backbone;
```

Comment:
if i remember correctly we were looking for quick wins to improve our server side throughput. switching from underscore to lodash improved things. i can't remember the exact numbers (maybe @ericherdzik can)